### PR TITLE
9.0 mig account payment purchase

### PR DESCRIPTION
**Pull request by cubells**
- Updated README
- Added account invoice method
- Updated procurement method
- Updated tests
- Updated views

cc @Tecnativa

### CODE REVIEW

**Comment by sbidoul** [comment]
Note the is work for the 10.0 migration in #352

**Comment by cubells** [comment]
@sbidoul 

Yes. I'll add some changes this pr.

Thanks.

File: account_payment_purchase/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Usage
+=====
+
+You are able to add a payment mode directly on a partner.
+This payment mode is automatically associated to the purchase order, then on
+related invoice.
+This default value could be change in a draft purchase or draft invoice.
+When you create a payment order, only invoices related to chosen payment mode
+are displayed.
+Invoices without any payment mode are displayed to.
```
Comment (luismontalba) [inline account_payment_purchase/README.rst]: +Invoices without any payment mode are displayed too.

**Review by luismontalba — APPROVED** [review]
Functionality tested on runbot (minor changes proposed in README.rst)

**Comment by pedrobaeza** [comment]
@cubells finish this PR with the comment from Luis, and don't let coverage to go down.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**